SpatialGDK/Source/SpatialGDK/Public/Utils/OpUtils.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include <WorkerSDK/improbable/c_worker.h>
+
+namespace SpatialGDK
+{
+	void FindFirstOpOfType(const TArray<Worker_OpList*>& InOpLists, const Worker_OpType OpType, Worker_Op** OutOp);
```

Comment:
Likewise here, don't think there should be an indent.

Reply:
fixed